cocos/scripting/js-bindings/manual/jsb_global.cpp

```diff
@@ -287,26 +290,26 @@ bool jsb_set_extend_property(const char* ns, const char* clsName)
 namespace {
 
     std::unordered_map<std::string, se::Value> __moduleCache;
-    #if (CC_TARGET_PLATFORM != CC_PLATFORM_OPENHARMONY) 
-        static bool require(se::State& s)
+    #if (CC_TARGET_PLATFORM == CC_PLATFORM_OPENHARMONY && SCRIPT_ENGINE_TYPE == SCRIPT_ENGINE_NAPI)
+        static bool run_script(se::State& s)
         {
             const auto& args = s.args();
             int argc = (int)args.size();
             assert(argc >= 1);
             assert(args[0].isString());
             return jsb_run_script(args[0].toString(), &s.rval());
         }
-        SE_BIND_FUNC(require)
+        SE_BIND_FUNC(run_script)
     #else
-        static bool run_script(se::State& s)
+        static bool require(se::State& s)
         {
             const auto& args = s.args();
             int argc = (int)args.size();
             assert(argc >= 1);
             assert(args[0].isString());
             return jsb_run_script(args[0].toString(), &s.rval());
         }
-        SE_BIND_FUNC(run_script)
+        SE_BIND_FUNC(require)
```

Comment:
What's the difference between `run_script` and `require` ?I saw the implementation of the binding code is the same.

Reply:
yes, fixed.